hbase-protocol-shaded/src/main/protobuf/HBase.proto

```diff
@@ -79,7 +79,6 @@ message RegionInfo {
   optional bool offline = 5;
   optional bool split = 6;
   optional int32 replica_id = 7 [default = 0];
-  optional string region_encoded_name = 8;
```

Comment:
This is not included in branch-2.2. So this is never released?

Reply:
Yes, so we should remove it before the 2.3.0 release.